editor.planx.uk/src/ui/Banner.tsx

```diff
@@ -41,30 +13,52 @@ interface BannerProps {
   children?: React.ReactNode;
 }
 
-function Banner(props: BannerProps) {
-  const classes = useClasses(props);
+const Root = styled(Box)(({ theme, color, bgcolor }) => ({
+  display: "flex",
+  justifyContent: "center",
+  textAlign: "left",
+  py: theme.spacing(6),
```

Comment:
Looks like padding can't be abbreviated in this way, CSS output in invalid (`py: 60px;`)

Suggestion to change to `padding: theme.spacing(6, 0)`